docs/user-guide/buckets.ipynb

```diff
@@ -131,15 +131,19 @@
    "source": [
     "xbins = sc.Variable(dims=['x'], unit=sc.units.m, values=[0.1,0.5,0.9])\n",
     "ybins = sc.Variable(dims=['y'], unit=sc.units.m, values=[0.1,0.3,0.5,0.7,0.9])\n",
-    "realigned = sc.realign(data, {'y':ybins,'x':xbins})\n",
-    "realigned"
+    "# TODO Would actually need to sort `data` first\n",
+    "begin = sc.Variable(dims=['y','x'], values=np.linspace(0,40,num=8).astype(np.int64).reshape(4,2))\n",
```

Comment:
Indeed this is temporary. See to do item in #1281: "Add to_buckets that accepts a dict of bin edges instead of indices. This would then sort the buffer (recursively) and determine indices. Similarely, splitting existing buckets by setting new bin edges (for existing or new dims) should be supportable in the same code."

Reply:
I think bin edges would be a good improvement here. However the current API has the side effect (if not originally intended) that it supports filtering.